caravel/assets/javascripts/explorev2/index.jsx

```diff
@@ -2,12 +2,31 @@ import React from 'react';
 import ReactDOM from 'react-dom';
 import ExploreViewContainer from './components/ExploreViewContainer';
 
+import { compose, createStore } from 'redux';
+import { Provider } from 'react-redux';
+
+import { initialState } from './stores/store';
+
 const exploreViewContainer = document.getElementById('js-explore-view-container');
 const bootstrapData = exploreViewContainer.getAttribute('data-bootstrap');
 
+
+import { exploreReducer } from './reducers/exploreReducer';
+import persistState from 'redux-localstorage';
+
+let enhancer = compose(persistState());
```

Comment:
i think i've seen this pattern in sql-lab too, would be great to add to redux utils.